### DISCUSSION TITLE
Could _typos.toml ignore folder?

### DISCUSSION THREAD

**Discussion by mobeicanyue** [crate-ci/typos#992] [Q&A]
Could _typos.toml ignore folder?

**Comment by epage** [comment]
Could you clarify what you mean?  Ideally, a reproduction case of "I assumed X would work and here is a config and a directory where it didn't work"

**Reply by mobeicanyue** [reply]
I want to ignore these directory and I didn't work.
![image](https://github.com/crate-ci/typos/assets/81098819/b01acfa5-07a1-4370-a3a0-56b51cf7f8ae)

**Reply by epage** [reply]
So something like:
```console
$ mkdir public
$ echo mispelled > public/file.txt
$ typos
error: `mispelled` should be `misspelled`
  --> ./public/file.txt:1:1
  |
1 | mispelled
  | ^^^^^^^^^
  |
$ echo -e "[files]\nextend-exclude = ['public']" > _typos.toml
$ typos
```

That seemed to work for me.

**Reply by mobeicanyue** [reply]
Thanks, It works, my bad.
